src/state/middleware/logging.ts

```diff
@@ -0,0 +1,19 @@
+import { Middleware } from '@reduxjs/toolkit'
+
+const log = (message: string, obj: unknown, featureFlag: boolean) => {
+  if (featureFlag) {
+    // eslint-disable-next-line no-console
+    console.log(message, obj)
```

Comment:
I believe we're trying to default to `console.info` instead of using `console.log`

Reply:
Thanks I've updated to info now